tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -107,3 +108,19 @@ def test_openpbs_driver_with_poly_example():
         "--enable-scheduler",
         "poly.ert",
     )
+
+
+@pytest.mark.timeout(180)
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("copy_poly_case")
+def test_openpbs_driver_with_poly_example_submit_fails(monkeypatch, caplog):
```

Comment:
What OpenPBS driver functionality is this testing?

Reply:
It was supposed to test that an exception in OpenPBS submit would crash ert, and propagate the error message to the user. This should maybe be a scheduler test instead of a openpbs driver test.